core/services/workflows/engine.go

```diff
@@ -836,7 +836,7 @@ func (e *Engine) workerForStepRequest(ctx context.Context, msg stepRequest) {
 
 	var stepStatus string
 	switch {
-	case errors.Is(capabilities.ErrStopExecution, err):
```

Comment:
Amended from https://github.com/smartcontractkit/chainlink/pull/13532/files

Reply:
Can you say more about why this change was necessary?

Reply:
Tagging @MStreet3 , because he made the original change, I just fixed it to work as intended.

Street's original change flipped it from 
```
// Not intended usage, see errors.Is doc
	case errors.Is(capabilities.ErrStopExecution, err):
```


```
// Intended usage
	case errors.Is(err, capabilities.ErrStopExecution):
```

Moving it to `case errors.Is(err, capabilities.ErrStopExecution):` caused tests to fail, so I changed it to what it is now.

Reply:
Maybe the problem is that we define "Is" ourselves? 

`func (e errStopExecution) Is(err error) bool {
	return strings.Contains(err.Error(), errStopExecutionMsg)
}`

Reply:
That self defined one ☝️  works as expected.